library/ImageConvert/Config/ImageConvertConfigInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Municipio\ImageConvert\Config;
+
+interface ImageConvertConfigInterface
+{
+    public function isEnabled(): bool;
+    public function imageDownsizePriority(): int;
+    public function mimeTypes(): array;
+    public function internalFilterPriority(): object;
+    public function fileNameSuffixes(): array;
+    public function maxImageDimension(): int;
+    public function intermidiateImageFormat(): array;
+    public function intermidiateImageQuality(): int;
```

Comment:
Optional, but may be good. To make the purpose of the returning methods obvious, I would prefix them all with `get`.
```suggestion
    public function isEnabled(): bool;
    public function getImageDownsizePriority(): int;
    public function getMmimeTypes(): array;
    public function getInternalFilterPriority(): object;
    public function getFileNameSuffixes(): array;
    public function getMaxImageDimension(): int;
    public function getIntermidiateImageFormat(): array;
    public function getIntermidiateImageQuality(): int;
```